xml_models/xml_models.py

```diff
@@ -222,8 +225,8 @@ def __new__(mcs, name, bases, attrs):
         return new_class
 
     def _get_xpath(cls, field_impl):
-        return property(fget=lambda cls: cls._parse_field(field_impl),
-                        fset=lambda cls, value: cls._set_value(field_impl, value))
+        return property(fget=lambda self: self._parse_field(field_impl),
```

Comment:
Here we are setting property for models that Inherit from xml_models.Model(or models has ModelBase as metaclass), the setter/getter function will not be called with now, the setter and getter function will be called on those instances later while we are serializing xml into python objects and using those data, and the `self` represents a particular instance object. 
Here the lambda expression just define the property, and the keyword `cls（should be self though)` in the lambda expression has nothing to do with the `MetaClass`. so i changed it to `self`. `underfined` situation will not happen